notices/rumbster-v1.1.0-NOTICE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think copyright could be copied from https://github.com/aesterline/rumbster#copyright

Reply:
thanks, i added it